src/pages/Offer/sections/offer/Discount.tsx

```diff
@@ -15,8 +15,9 @@ import { OfferData } from 'src/containers/OfferContainer'
 import * as Yup from 'yup'
 import { RedeemCodeMutation } from '../../containers/RedeemCodeMutation'
 
-const DiscountButtonLink = styled('button')({
+const DiscountButtonLink = styled('a')({
```

Comment:
this seems odd to me, any particular reason?

Reply:
Ah, i saw the commit message. When does it look terrible? I think that would be the case for virtually all buttons if that's the case no?

Reply:
I mostly used this solution to get rid of the default button styling that web browsers sometimes ship. We decided that we wanted this button to instead be a link (https://trello.com/c/yVM1vUKl/191-web-replace-rabattkod-button-with-text), and this just makes it so that some browsers don't display it as a button 🙂

Reply:
Firefox rendered it as a good old gray square which felt a bit off

Reply:
The styling on the other buttons fully override the default button styling afaik, which makes this a non-issue for those buttons :slig